app/app.go

```diff
@@ -1335,6 +1335,7 @@ func (d *Deployments) GetDeploymentForDeviceWithCurrent(ctx context.Context, dev
 			l := log.FromContext(ctx)
 			l.Warn(errors.Wrap(err, "failed to trigger a device reindex"))
 		}
+		var newArtifactAssigned bool
```

Comment:
Is this `var` in the right place? Why is it just before the `return nil, nil`?

Reply:
it's at the beginning of the method; maybe it's not that well visible in the unified view?

Reply:
Yep, it was a glitch in the github view.